static/js/containers/pages/CheckoutPage_test.js

```diff
@@ -379,11 +379,7 @@ describe("CheckoutPage", () => {
       const expected = {}
       for (const runId of item.run_ids) {
         for (const course of item.courses) {
-          for (const run of course.courseruns) {
-            if (run.id === runId) {
-              expected[course.id] = run.id
-            }
-          }
+          expected[course.id] = course.next_run_id
```

Comment:
I think this could use a test beyond the existing test case

Reply:
I thought about adding another test, but we don't have course runs sorted in our basket from the factory. I can't generate expected data without manually writing the actual code what we have in `calcSelectedRunIds()`